Automattic/jetpack
# Fix My Jetpack TOS `<p>` cannot be descendant of `<p>` warning

Fixes p1713873448166689-slack-C02LK1W8T4Z

## Proposed changes:

* This PR fixes two minor issues:
  * The TOS text (a `p` element itself) was being rendered within a `p` tag, causing it to throw a warning in the console.
  * The font-size of the TOS was slightly bigger than what it should be

![image](https://github.com/Automattic/jetpack/assets/5714212/6eaa2000-bd92-49aa-968f-2f80ac916da5)
<img width="1099" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/a32dc640-509f-4046-9353-c87132e421b0">


### Other information:


## Jetpack product discussion


N/a

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Visit My Jetpack, dismiss the Welcome Banner and refresh the page
* After refreshing and with the console tab open, confirm that you don't see any other warnings related to that issue, and confirm that the TOS font-size is `14px`
